cypress/support/commands.js

```diff
@@ -189,20 +197,30 @@ Cypress.Commands.add('createSubmission', (data, context) => {
 				cy.get('input[type=file]').attachFile(
 					{fileContent, 'filePath': file.fileName, 'mimeType': 'application/pdf', 'encoding': 'base64'}
 				);
-				var $row = cy.get('a:contains("' + file.fileName + '")').parents('.listPanel__item');
+				cy.wait('@fileUploaded').its('status').should('eq', 200);
+				const $row = cy.get('a:contains("' + file.fileName + '")').parents('.listPanel__item');
+				cy.wait(1000);
```

Comment:
Is this wait a critical part of the test being improved? It sounds from the original report like the defined route (`@fileUploaded`) might be enough... I consider every one of our `cy.wait` calls a FIXME, and there are lots!

Reply:
Unfortunately it was needed, I guess some text is appearing before the handler is attached to the button, so the click fails, I probably just can reduce the value to 200ms.

I think that a proper fix will have to wait for a signal from `Vue`, perhaps something generic using the [lifecycle events](https://vuejs.org/guide/essentials/lifecycle.html#lifecycle-diagram)... Like checking for a counter ruled by "created increases a counter, mounted decreases it"... But then, there's also jQuery, timers, etc. 😣

Reply:
OK! No need to drop it from 1000 to 250; let's deal with these systematically later, possibly when we have time to revisit our seeding/testing strategy.